### ISSUE TITLE
browserify the src is unsuccessful

### ISSUE THREAD

**Issue by neoroma** [christianalfoni/flux-angular#60]
Executing the command 

```
browserify .\node_modules\flux-angular\src\flux-angular.js -t [ babelify --presets [ es2015 ]
```

gives an error

```
SyntaxError: Unexpected token (3:6) while parsing C:\dev\projects\face\node_modules\flux-angular\src\flux-angular.js while parsing file: C:\dev\projects\face\node_modules\flux-angular\src\flux-angular.js ... 
```

babelify@7.2.0
browserify@13.0.0

(Windows 10)

**Comment by jrust** [comment]
Looks like you're missing an end bracket.  This should work:
```
browserify .\node_modules\flux-angular\src\flux-angular.js -t [ babelify --presets [ es2015 ] ]
```

**Comment by neoroma** [comment]
Yes, you are right but I'm getting the same error while having flux-angular as require in my app, while browserify bundling.

**Comment by jrust** [comment]
Can you provide more code and details to reproduce the second problem?

**Comment by neoroma** [comment]
Ok I've got in my package.json
```
"devDependencies": {
    "babel-preset-es2015": "^6.3.13",
    "babelify": "^7.2.0"
  },
  "dependencies": {
    "angular": "^1.4.8",
    "flux-angular": "^3.0.0"
  }
```
and browserify installed globally.

Index.js file code is simple :
```
const angular = require('angular')
require('flux-angular')
angular.module('example', [flux])
```

So when I execute 
```
browserify index.js -t [ babelify --presets [ es2015 ] ]  
```

getting the error

```
SyntaxError: Unexpected token (3:6) while parsing /Users/jekaterinasheremetjeva/Documents/workspace/fa/node_modules/flux-angular/src/flux-angular.js while parsing file: /Users/jekaterinasheremetjeva/Documents/workspace/fa/node_modules/flux-angular/src/flux-angular.js
    at DestroyableTransform.end [as _flush] (/usr/local/lib/node_modules/browserify/node_modules/insert-module-globals/index.js:96:21)
    at DestroyableTransform.<anonymous> (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_transform.js:123:12)
    at DestroyableTransform.g (events.js:260:16)
    at emitNone (events.js:67:13)
    at DestroyableTransform.emit (events.js:166:7)
    at prefinish (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_writable.js:501:12)
    at finishMaybe (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_writable.js:509:7)
    at endWritable (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_writable.js:521:3)
    at DestroyableTransform.Writable.end (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_writable.js:486:5)
    at DestroyableTransform.onend (/usr/local/lib/node_modules/browserify/node_modules/readable-stream/lib/_stream_readable.js:545:10)
```

**Comment by jrust** [comment]
Hmm, strange.  I fiddled a bit thinking maybe the `main` entry in package.json from flux-angular should be the release version but that didn't solve it.  Re-opening in the hopes someone will come across a solution.

**Comment by neoroma** [comment]
Any news on that?

**Comment by jrust** [comment]
I gave various things a try and couldn't figure out anything.  If you're able to look that'd be great.

**Comment by jrust** [comment]
Fixed by 52ea2ed7a98946a943cbbb89f89eb89d7effdbc3
